docs/artifacts/one-version-two-spellings.html

```
@@ -0,0 +1,601 @@
+<title>One Version, Two Spellings</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)